index.html

```diff
@@ -19,12 +19,19 @@
   <nav>
     <a id="logo" href="./index.html" class="a-button">
       <div><img src="dist/assets/squares-logo.png"></div>
-      <div><span>AdoptOpenJDK</span></div>
+      <div><span>AdoptOpenJDK - GetOpenJDK</span></div>
     </a>
   </nav>
 
   <main class="grey-bg home-main">
-    <p id="intro">Java™ is a popular open-source programming language.  It is used extensively, from applications running large corporate systems to small computers running personal games.  The code for Java is open source and available at <a href="http://openjdk.java.net" target="_blank">OpenJDK™</a>.</p>
+    <p class="intro">
+      Java™ is the world's leading programming language and platform. The code for Java is <a href="http://openjdk.java.net/legal/gplv2+ce.html">open source</a> and available at
+      <a href="http://openjdk.java.net" target="_blank">OpenJDK</a>™. AdoptOpenJDK provides prebuilt OpenJDK binaries
+      from a fully open source set of <a href="https://github.com/AdoptOpenJDK/openjdk-build">build scripts</a> and infrastructure.
+    </p>
+    <p class="intro">
+      <strong>NOTE:</strong> AdoptOpenJDK binaries are <strong>not</strong> professionally supported, see <a href="./support.html">Support</a> for details.
```

Comment:
Instead of `<strong>NOTE:</strong>`, we could use:

`<i class="fa fa-info-circle" aria-hidden="true"></i>`